sympy/functions/combinatorial/numbers.py

```diff
@@ -1923,6 +1935,21 @@ def nT(n, k=None):
     >>> nT('1'*4)
     5
 
+    Another way of stating the above is that the number of partitions
+    of `n` identical items into `k` or fewer parts is just the number
+    of partititions of `n + k` identical items into `k` parts,
+
+    >>> nT(4 + 2, 2) == nT(4, 0) + nT(4, 1) + nT(4, 2)
+    True
+
+
+    Another way of stating the above is that the number of partitions
```

Comment:
Is this text repeated for some particular purpose?

Reply:
I will change it.

Reply:
@jksuom  I removed those extra text.